### PR TITLE
DSD-899: Checkbox border radius

### PR DESCRIPTION
**Pull request by bigfishdesign13**
Fixes JIRA ticket [DSD-899](https://jira.nypl.org/browse/DSD-899)

## This PR does the following:

- Updates the `Checkbox` component to use "3px" for the border radius.

## How has this been tested?

- n/a

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

File: src/theme/components/checkbox.ts [hunk 1]
```diff
@@ -87,7 +87,7 @@ const sizes = {
     // Controls the width/height of the checkbox itself.
     control: {
       ...checkboxRadioControlSize,
-      borderRadius: "sm",
+      borderRadius: "3px",
```
Comment (EdwinGuzman) [inline src/theme/components/checkbox.ts]: We can create custom radius values. Maybe this should be a custom name as well.

- Here's the theme file: https://github.com/NYPL/nypl-design-system/blob/development/src/theme/foundations/radii.ts
- And how it's used: https://github.com/NYPL/nypl-design-system/blob/development/src/theme/components/button.ts#L67

**Review by EdwinGuzman — APPROVED** [review]
Not really a request change but something to point out in case you want to do it.

File: src/theme/components/checkbox.ts (hunk 1 above)
Comment (bigfishdesign13) [reply src/theme/components/checkbox.ts]: Good point.  I made the change to use a custom radius value.

**Review by EdwinGuzman — APPROVED** [review]

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr934-6aoaykzgisefjwascy3kmwvh5rsqu94f.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/624b43f6d9027707d3f354f6

**Pull request merged**